### PR TITLE
[api] Bump to API level 3. Contributes to JB#33571

### PR DESCRIPTION
**Pull request by jpetrell**

### CODE REVIEW

File: patterns/sailfish-api-3.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - qt5-qtdeclarative-import-models2
+    - qt5-qtmultimedia
+    - qt5-qtmultimedia-plugin-audio-pulseaudio
+    - qt5-qtpositioning
+    - qt5-plugin-imageformat-gif
+    - qt5-plugin-imageformat-ico
+    - qt5-plugin-imageformat-jpeg
+    - qt5-qtsvg-plugin-imageformat-svg
+    - libsailfishapp
+    - SDL2
```
Comment (saukko) [inline patterns/sailfish-api-3.yaml:29]: Is SDL2 enough itself? There is quite a bit of subpackages related to SDL2.

**Comment by saukko** [comment]
One small comment but probably doesn't need changes thus :+1:

File: patterns/sailfish-api-3.yaml (hunk 1 above)
Comment (jpetrell) [inline patterns/sailfish-api-3.yaml:29]: Most subpackages are examples or debug libraries, I read them through earlier. But I'll cross-check with harbour white list again, I may be missing something.

Comment (jpetrell) [inline patterns/sailfish-api-3.yaml:29]: Yes, SDL2_gfx, SDL2_image, SDL2_mixer, SDL2_net, SDL2_ttf are also needed.

Comment (jpetrell) [inline patterns/sailfish-api-3.yaml:29]: Fixed. Embarrassing I didn't notice those.

**Comment by saukko** [comment]
:+1:

**Pull request merged**